File: Modul-2/soal-4/PRAK204_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
float phi = 22.0/7.0;
float r, tinggi, volume, lpermukaan, kll;
printf("Input jari-jari dan tinggi : \n");
scanf("%f %f", &r, &tinggi);
volume = phi * r * r * tinggi;
lpermukaan = (2 * phi * r * tinggi) + (2 * phi * r * r);
kll = 2 * phi * r;
printf("Volume = %.2f\n", volume);
printf("Luas = %.2f\n", lpermukaan);
printf("Keliling = %.2f", kll);
return 0;
}

File: Modul-2/soal-4/PRAK204_2410817210017_OrlandoSugian.py

phi = 22/7
print("Input jari-jari dan tinggi : ")
my_input = []
while(len(my_input) < 2):
    x = input()
    x = x.split()
    my_input += x
r = float(my_input[0])
tinggi = float(my_input[1])
volume = float(phi * r ** 2 * tinggi)
lpermukaan = float((2 * phi * r * tinggi) + (2 * phi * r**2))
kll = float(2 * phi * r)
print(f"Volume = {volume:.2f}")
print(f"Luas = {lpermukaan:.2f}")
print(f"Keliling = {kll:.2f}")